conjur/cli.py

```diff
@@ -706,33 +707,38 @@ def run_action(resource, args):
 
         credential_provider, _ = CredentialStoreFactory.create_credential_store()
         # pylint: disable=no-else-return,line-too-long
+        if resource == 'logout':
```

Comment:
why was this placed here instead of below ? Does this make a difference?

Reply:
where below?

Reply:
where it was